src/main/java/com/nha/abdm/wrapper/common/dataPackaging/encryption/EncryptionController.java

```diff
@@ -0,0 +1,175 @@
+/* (C) 2024 */
+package com.nha.abdm.wrapper.common.dataPackaging.encryption;
```

Comment:
any reason to have this in common package and not in hip hrp?

Reply:
general idea is to have those things in common package which can be used across hip and hiu modules